app/helpers/search_helper.rb

```diff
@@ -0,0 +1,7 @@
+module SearchHelper
+
+  def content_search type,text
```

Comment:
[IMO] This logic is not suitable to put in a helper. Please put it in controller/model